catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'frontend-app-shell'
+  description: "A Piral shell application for use with Open edX MFEs."
+  links:
+    - url: "https://github.com/ichintanjoshi/frontend-app-shell"
+      title: "frontend-app-shell"
+      icon: "GitHub"
+  annotations:
+    openedx.org/arch-interest-groups: ""
+    openedx.org/release: "main"
+spec:
+  owner: group:openedx-unmaintained
```

Comment:
@arbrandes This seems like it's new enough that we should have a maintainer.  Are you the right person to be the maintainer for this repo?

Reply:
Yeah, sign me up!